### PR TITLE
Bar docs for 0.9.2

### PR DESCRIPTION
**Pull request by MitchellNZ**
@stsrki this is still a WIP. 
Feedback welcome!

Docs for new Bar features added by #1042

### CODE REVIEW

File: docs/_docs/components/bar.md [hunk 1]
```diff
@@ -165,7 +158,135 @@ Right-aligned vertical bar example:
...
+    ...
+  </BarBrand>
+  <BarToggler Mode="BarTogglerMode.Popout" />
+  <BarMenu>
+    ...
+  </BarMenu>
+</Bar>
+```
+
+**3. Extneral** (controlled by Top Bar)
```
Comment (stsrki) [inline docs/_docs/components/bar.md:214]: Small typo :)

File: docs/_docs/components/bar.md [hunk 2]
```diff
@@ -165,7 +158,135 @@ Right-aligned vertical bar example:
...
+    <LayoutSiderContent>
+      <Bar @ref="sidebar" Mode="BarMode.VerticalInline">
+        ...
+      </Bar>
+    </LayoutSiderContent>
+  </LayoutSider>
+  <Layout>
+    <LayoutHeader Fixed="true">
+      <Bar Mode="BarMode.Horizontal">
+        <BarToggler Bar="sidebar" />
```
Comment (stsrki) [inline docs/_docs/components/bar.md:237]: I like this approach :)

Comment (stsrki) [reply docs/_docs/components/bar.md:237]: PS. Try to have `@` whenever there is variable used. eg. `<BarToggler Bar="@sidebar" />`. I found it a good convention.

File: docs/_docs/components/bar.md (hunk 1 above)
Comment (MitchellNZ) [reply docs/_docs/components/bar.md]: Thanks for all the typo fixes! 
I was doing this on my laptop.. not used to the keyboard 😂

Comment (stsrki) [reply docs/_docs/components/bar.md]: Haha np, you did most of the things anyways, so I should be the one giving the thanks :)

**Pull request merged**